src/styles/general/_variables.scss

```diff
@@ -2,6 +2,9 @@
 $color-black: #000;
 $color-white: #fff;
 $color-blue: #007bd3;
+$color-gray: #f5f5f5;
+$color-dark-gray: #2c2c2c;
```

Comment:
I would try using some numbering for shades since this will go wrong with more shades dark, darker, more darker... 😄